### PR TITLE
Tests

### LINKED ISSUE #2: Update all tests to use apex code

### PR DESCRIPTION
**Pull request by nchursin**
Resolves #2

### CODE REVIEW

File: tests/primitive_type-test.js [hunk 1]
```diff
@@ -1,60 +1,59 @@
...
+
+describe('primitiveType', () => {
+  it('boolean', () => {
+    expect(Parser.parse('boolean', (parser) => parser.primitiveType())).toEqual({
+      type: 'PRIMITIVE_TYPE',
+      value: 'boolean',
+    })
+  })
+
+  it('char', () => {
```
Comment (dmitrii-nikitin) [inline tests/primitive_type-test.js]: we should remove char and byte data types as it is not supported in apex

Comment (dmitrii-nikitin) [reply tests/primitive_type-test.js]: as well as short and float

File: tests/enum_declaration-test.js [hunk 2]
```diff
@@ -1,206 +1,191 @@
...
 
-  it("enumConstants with comma && enumBodyDeclarations", () => {
-    expect(
-      Parser.parse("enum A{B,; void a() {}}", parser =>
-        parser.enumDeclaration()
-      )
-    ).toEqual({
-      type: "ENUM_DECLARATION",
+  it('enumConstants with comma && enumBodyDeclarations', () => {
+    expect(Parser.parse('enum A{B,; void a() {}}', (parser) => parser.enumDeclaration())).toEqual({
```
[the remark this answers is not in the document]
Comment (nchursin) [reply tests/enum_declaration-test.js]: no, not really. I'll update the rule

File: tests/block-test.js [hunk 3]
```diff
@@ -1,315 +1,279 @@
...
+  })
 
-  it('System.out.println("please work")', () => {
-    expect(
-      Parser.parse('{ System.out.println("please work"); }', parser =>
-        parser.block()
-      )
-    ).toEqual({
-      type: "BLOCK",
+  it("System.out.println('please work')", () => {
```
[the remark this answers is not in the document]
Comment (nchursin) [reply tests/block-test.js]: will do

File: tests/primitive_type-test.js (hunk 1 above)
Comment (nchursin) [reply tests/primitive_type-test.js]: yup, will do

**Pull request merged**